src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -3,7 +3,7 @@ import { useQuery } from '@tanstack/react-query';
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useLocation } from 'react-router-dom';
-import { fetchResults, FetchResultsParams, ResultParam, SortOrder } from '../../../api/searchApi';
+import { FetchResultsParams, ResultParam, SortOrder, fetchResults } from '../../../api/searchApi';
```

Comment:
🤔

Reply:
`"editor.codeActionsOnSave": {
    "source.organizeImports": "explicit"
  },`
  i `settings.json` 😅

Reply:
(Ikke at du trenger å gjøre noe for min del altså. Bare en kuriositet)

Reply:
_What is the best front-end code editor, and why is it VS Code?_

Reply:
case-sensitive vs case-insensitiv sortering, faktisk 😅 vi burde egentlig ha en praksis på hva vi ønsker å bruke da, men usikker på hva det skal være. Virker som vscode er case-sensitive, mens IntelliJ(?) ikke er det. Har ikke vært noe problem før, siden både jeg og Isar har brukt vscode.
Tror egentlig jeg liker case-insensitive best 🤔

Reply:
Har ingen sterke meninger rundt dette

Reply:
bare å grave litt i dette her om noen har lyst! Eller blir det en del teite diffs som dette fremover

Reply:
Kan sjekke om IntelliJ har en setting for dette 👍